kanzucode/angie-server
# Ref-027 As a cohort leader, I can see the percentage of those that attended [total number of people who attended MC / total number of people in MCs] for my cohort

**What does this PR do?**

- This PR adds a feature that allows cohort leaders to see the average percentage attendance of MCs in their cohort over a month.

- This PR also fixes the report display under the Group Details page. Cohort leaders can also see the MC reports associated with their cohort.

**Description of tasks?**

- Cohort leaders can now see the average percentage attendance for MCs under their cohort, in the current month, and the reports of MCs under their cohort.

**How to manually test this?**

- Under the `Group Details` the percentage attendance for the current month should be visible. And, under the `Reports` tab, Cohort leaders should see the reports of their MCs.

[Link to frontend PR](https://github.com/kanzucode/angie-client/pull/48)
